lms/static/scripts/frontend_apps/components/Dialog.js

```diff
@@ -132,7 +134,7 @@ export default function Dialog({
               <Button
                 className="Button--cancel"
                 onClick={onCancel}
-                label="Cancel"
+                label={cancelLabel}
```

Comment:
Adding a way to override this label because  "Cancel" does not make sense when presenting an error when the only action is to close the dialog.